### ISSUE TITLE
[BUG] does not overlay full screen Remote Desktop on MacOS 

### ISSUE THREAD

**Issue by ligboo** [hovancik/stretchly#1060]
Running Stretchly v1.8.1 on MacOS Catalina 10.15.7.

With this latest version, the stretchly window stopped overlaying my full screen Microsoft Remote Desktop window.

Used to work fine.

**Comment by JorisDebonnet** [comment]
Yeah, I noticed something changed in how much it covers indeed. The way I noticed it was that the context menu (right click) in the system tray (on Windows) appears over the Stretchly window now, whereas in the past that menu used to go behind it. (I actually liked that change, but of course it's less desired if work apps such as Remote Desktop remain on top.)

**Comment by hovancik** [comment]
Not sure what version it has changed, but what @JorisDebonnet described is what I tried to achieve, as it makes sense that global menus should be accessible. 

I am setting `pop-up-menu`in `level`  https://www.electronjs.org/docs/latest/api/browser-window#winsetalwaysontopflag-level-relativelevel

**Comment by axelcostaspena** [comment]
Same with my full screen [WebStorm](https://www.jetbrains.com/es-es/webstorm/)

**Comment by bdrum** [comment]
Thanks for the app, that's great!
Same issue for Windows Remote Desktop app (v1.12.0)

**Comment by hovancik** [comment]
Hi, is still still issue with latest Stretchly? What are your Stretchly preferences so that I can try to reproduce?
